api/core/agent/fc_agent_runner.py

```diff
@@ -17,28 +17,29 @@
     ToolPromptMessage,
     UserPromptMessage,
 )
+from core.prompt.agent_history_prompt_transform import AgentHistoryPromptTransform
 from core.tools.entities.tool_entities import ToolInvokeMeta
 from core.tools.tool_engine import ToolEngine
 from models.model import Message
 
 logger = logging.getLogger(__name__)
 
 class FunctionCallAgentRunner(BaseAgentRunner):
+    _query: str = None
+    _current_thoughts: list[PromptMessage] = []
```

Comment:
It has the same issue, maybe we need to set a None as the default.

Reply:
Here is an example, I just began a new conversation, however, there are 3 items in this array
<img width="776" alt="image" src="https://github.com/langgenius/dify/assets/45712896/001b3375-02e3-4fcc-aac0-39132de9456e">
<img width="1259" alt="image" src="https://github.com/langgenius/dify/assets/45712896/e2f5a432-99a1-4225-a8c3-2059aa471dec">

Reply:
fixed